python/semantic_kernel/contents/streaming_annotation_content.py

```diff
@@ -5,7 +5,9 @@
 from xml.etree.ElementTree import Element  # nosec
 
 from pydantic import Field
+from pydantic_settings import SettingsConfigDict
```

Comment:
this is for Pydantic Settings, should be:
```suggestion
from pydantic import ConfigDict
```